lib/eclipse/include/opm/parser/eclipse/Units/Units.hpp

```diff
@@ -167,7 +167,7 @@ namespace Opm {
         constexpr const double degCelsiusOffset = 273.15; // offset for the °C -> K conversion
 
         constexpr const double degFahrenheit = 5.0/9; // scaling factor °F -> K
-        constexpr const double degFahrenheitOffset = 255.37; // offset for the °C -> K conversion
+        constexpr const double degFahrenheitOffset = 459.67*5.0/9.0; // °F -> K offset (i.e. what's 0 °F?)
```

Comment:
I guess the trailing `5.0/9.0` could be expressed with the `degFahrenheit` symbol?

Reply:
good idea. I'll change this.